README.md

```diff
@@ -45,6 +46,13 @@ $ docker run  -p 3000:3000  transfer-server-validator
 $ docker run -e DOMAIN=http://<yourdomain.com transfer-server-validator
 ```
 
+## Run a specified project via Docker
+
+```
+$ docker build -t transfer-server-validator .
+$ docker run -p 3000:3000 -e PROJECT=DIRECT-PAYMENT transfer-server-validator
```

Comment:
Lets use underscore instead of hyphen so we can use the projects as keys on JS objects easier